crowbar_framework/app/assets/stylesheets/content/footer.css.scss

```diff
@@ -1,6 +1,8 @@
 footer {
-  text-align: center;
+  color: #999;
```

Comment:
Color literals like `#999` should only be used in variable declarations; they should be referred to via variable everywhere else.